nautobot_device_lifecycle_mgmt/filters.py

```diff
@@ -1,31 +1,29 @@
 """Filtering implementation for the Lifecycle Management plugin."""
 import datetime
+
 import django_filters
 from django.db.models import Q
-
-from nautobot.dcim.models import DeviceType, Platform, Device, DeviceRole, InventoryItem, Region, Site
+from nautobot.dcim.models import Device, DeviceRole, DeviceType, InventoryItem, Platform, Region, Site
+from nautobot.extras.filters import CustomFieldModelFilterSet, StatusFilter, StatusModelFilterSetMixin, TagFilter
 from nautobot.extras.models import Tag
-from nautobot.extras.filters import (
-    StatusModelFilterSetMixin,
-    CustomFieldModelFilterSet,
-    TagFilter,
-    StatusFilter,
-)
+from nautobot.utilities.filters import BaseFilterSet
```

Comment:
In 1.4, most filtersets should probably inherit from `nautobot.extras.filters.NautobotFilterSet` as it combines `(BaseFilterSet, CreatedUpdatedFilterSet, RelationshipModelFilterSet, CustomFieldModelFilterSet)`